Specs/Scene/BillboardCollectionSpec.js

```diff
@@ -74,21 +80,19 @@ defineSuite([
         billboards = new BillboardCollection();
 
         if (!greenImage) {
-            greenImage = new Image();
-            greenImage.src = './Data/Images/Green.png';
-
-            blueImage = new Image();
-            blueImage.src = './Data/Images/Blue.png';
-
-            whiteImage = new Image();
-            whiteImage.src = './Data/Images/White.png';
-
-            largeBlueImage = new Image();
-            largeBlueImage.src = './Data/Images/Blue10x10.png';
-
-            waitsFor(function() {
-                return greenImage.complete && blueImage.complete && whiteImage.complete && largeBlueImage.complete;
-            }, 'Load .png file(s) for billboard collection test.', 3000);
+            return when.join(
```

Comment:
Putting this in `beforeEach` is a workaround.  It should go in `beforeAll` without `if (!greenImage) {`.  Do you think that will work now in Jasmine 2.0?